Matrix.py

```diff
@@ -0,0 +1,120 @@
+class Matrix:
+    # in this program Matrix is a tuple of tuples and the inside tuples must be as long as the major ine is.
+    # data is the Matrix (tuple) received from the user
+    def __init__(self, data):
+        counter = 0
+        if type(data) is tuple:
+            for tup in data:
+                if len(tup) == len(data) and type(tup) is tuple:
+                    counter += 1
+        if counter == len(data):
+            self.data = data
```

Comment:
I think that suppose to be inside the **if** of line 6

Reply:
in the last review of natsala13 i was asked to do it with counter instead of build it in the if of line 6